src/selectField/index.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { Ref } from 'react'
+import cc from 'classcat'
+import { ChevronIcon } from 'icon'
+import { color } from '_utils/branding'
+import style from 'selectField/style'
+
+export interface SelectFieldProps {
+  readonly name: string
+  readonly className?: Classcat.Class
+  readonly value: string
+  readonly optGroupLabel: string
+  readonly options: Object[]
+  readonly onChange?: (obj: OnChangeParameters) => void
+  readonly forwardedRef?: Ref<HTMLDivElement>
+}
+
+const SelectField = ({
+  name,
+  className,
+  value,
+  optGroupLabel,
+  options,
+  onChange,
+  forwardedRef,
+}: SelectFieldProps) => {
+  const baseClassName = 'kirk-selectField'
+
+  return (
+    <div className={cc([baseClassName, className])} ref={forwardedRef}>
+      <select
+        name={name}
+        onChange={event => onChange({ name, value: event.target.value })}
+        defaultValue={value}
+      >
+        <optgroup label={optGroupLabel}>
```

Comment:
I guess there should be a `optGroups` prop, an array of objects each with their own label and options (and label should be optional, no `optGroup` tag if no label).

Reply:
Agree about `optGroup` not required, I added it to manage the fact there is no placeholder and no label bellow select in Pixar library.

Reply:
At the end I added `ariaLabel` prop to keep accessibility if no `<label>` tag linked ton the `select` tag.